### DISCUSSION TITLE
I try to use Calendar by CDN but some props don´t work as disabledDays, disabledDates and others, if someone has a example to help me!

### DISCUSSION THREAD

**Discussion by Rodo27** [primefaces/community#1349] [PrimeVue] [answered]
<script src="https://unpkg.com/vue@3/dist/vue.global.js"></script>
        <script src="https://unpkg.com/primevue/core/core.min.js"></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/primevue/3.49.1/calendar/calendar.min.js"></script>
        <div id="app">
            <p-datepicker v-model="date" disabledDays="[1]"></p-datepicker>
            <br /><br />
            
        </div>

        <script>
            const { createApp, ref } = Vue;

            const app = createApp({
                setup() {
                    const date = ref();
                    return {
                        date,

                    };
                },
                components: {
                    "p-datepicker": primevue.calendar
                }
            });

            app.use(primevue.config.default);
            //app.component("p-datepicker", primevue.calendar);

            app.mount("#app");
        </script>

**Comment by tugcekucukoglu** [comment, marked as answer]
Could you try to use disabledDays as `disabled-days=""`? It seems to be working https://stackblitz.com/edit/web-platform-qq5ahr?file=index.html

**Reply by Rodo27** [reply]
Thanks! it´s works!

So why in the documentation is set like disabledDays?

There is a site to see the correct prop like use by CDN?

Another props has this structure?

like
disabledDates as disabled-dates 
disabledDays as disabled-days
